### PR TITLE
Update build.gradle.kts

### PR DESCRIPTION
**Pull request by Ramseyxlil**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -10,7 +10,6 @@
 import org.gradle.internal.os.OperatingSystem
 import java.text.SimpleDateFormat
 import java.util.Date
-
```
Comment (davidhutchens) [inline build.gradle.kts:13]: I don't see the point in removing this line.

**Review by davidhutchens — CHANGES_REQUESTED** [review]
I don't understand the purpose of this PR. Please provide more information if there is an actual purpose. Otherwise, I would recommend closing without merging.

**Comment by MarcinOrlowski** [comment]
I think there's no purpose here. It's just a garbage PR which should be rejected. OP should have no problem sending new PR if there's any contribution he tried but failed to provide (yet his for indicates no work).

**Comment by BFH-ktt1** [comment]
Closing this PR as no contribution

**Pull request closed without merging**